packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -862,6 +864,8 @@ const ruleset$1 = {
             description: "All schemas should have a description or title.",
             message: "Schema should have a description or title.",
             severity: "warn",
+            disableForTypeSpec: true,
+            disableForTypeSpecReason: "Covered by TSP's 'documentation-required' rule.",
```

Comment:
```suggestion
            disableForTypeSpecReason: "Covered by TSP's '@azure-tools/typespec-azure-core/documentation-required' rule.",
```

not sure how much this matter but if you want to be accurate in the reference this is the full name of the rule